src/quo2/components/drawers/drawer_buttons/style.cljs

```diff
@@ -26,18 +27,19 @@
    :padding-horizontal      20
    :border-top-left-radius  20
    :border-top-right-radius 20
+   :backdrop-filter         "blur(20px)"
```

Comment:
could you elaborate?

Reply:
This is for blur effect in the bottom drawer buttons.

Reply:
Yes, I think `backdrop-filter` should not be supported properly in React Native. I'll replace it with blur directly.